labrad/backend.py

```diff
@@ -45,6 +47,13 @@ def disconnect(self):
             self._disconnect()
             self.connected = False
 
+    def spawn(self):
+        """Start a new independent backend connection to the same manager."""
+        inst = self.__class__(name=self.name)
```

Comment:
Yes, this creates a new instance of the class. Doing it this way means it will work for any subclass of `BaseConnection`, because we call the constructor for whatever the runtime class of `self` actually is.